src/docs/markdown/caddyfile/options.md

```diff
@@ -146,62 +168,77 @@ Customizes the named logger. The name can be passed to indicate a specific logge
 
 
 ##### `grace_period`
-Defines the grace period for shutting down HTTP servers during config reloads. If clients do not finish their requests within the grace period, the server will be forcefully terminated to allow the reload to complete and free up resources.
+Defines the grace period for shutting down HTTP servers during config reloads. If clients do not finish their requests within the grace period, the server will be forcefully terminated to allow the reload to complete and free up resources. By default, no grace period is set.
```

Comment:
```suggestion
Defines the grace period for shutting down HTTP servers during config reloads. If clients do not finish their requests within the grace period, the server will be forcefully terminated to allow the reload to complete and free up resources.
```

Same here, not sure how much I love duplicating docs, makes them harder to maintain.

Reply:
We're not duplicating in this case, because it's not mentioned otherwise that this _isn't_ turned on by default.